README.md

```diff
@@ -32,16 +32,25 @@ The module expect the Bridgecrew platform API token to be supplied.
 On Windows:
 
 ```powershell
-$env:TF_VAR_apitoken="your-platform-token"
+$env:TF_VAR_api_token="your-platform-token"
 ```
 
 On *nix:
 
 ```shell
-export TF_VAR_apitoken="your-platform-token"
+export TF_VAR_api_token="your-platform-token"
 ```
 
-Or you can leave it to be prompted at your console.
+Or you can leave it blank to be prompted at your console.
+
+This module is supported for deployment in the following AWS regions:
+
+* All US regions
+* eu-west-1
```

Comment:
can we add a validation rule on those regions?
https://www.terraform.io/docs/configuration/variables.html#custom-validation-rules

Reply:
A lot of customers still use v0.12.